src/ontology/mondo-edit.obo

```diff
@@ -582793,6 +582788,20 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-01T21:00:19Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7652" xsd:anyURI
 
+[Term]
+id: MONDO:1040033
+name: congenital muscular dystrophy without intellectual disability
+subset: ordo_disorder {source="Orphanet:370980"}
+subset: otar {source="MONDO:OTAR"}
+synonym: "CMD without intellectual disability" EXACT [Orphanet:370980]
```

Comment:
please add the definition from Orphanet: https://www.orpha.net/en/disease/detail/370980?name=370980&mode=orpha